modelica_builder/modelica_project.py

```diff
@@ -86,17 +87,41 @@ def __init__(self, package_file):
         self.root_directory = Path(package_file).parent
         self.file_types = ['.mo', '.txt', '.mos', '.order', ]
         self.file_types_to_skip = ['.log', '.mat', ]
+
         # skip some specific files that software may create that are not needed to be
         # sent around with the modelica project.
-        self.file_names_to_skip = ['analysis_variables.csv', 'analysis_name.txt', '.DS_Store', ]
+        self.file_names_to_skip = ['.DS_Store', '.mpignore']
+
+        # Keep track of the files that have been ignored.
+        self.mpignore_files = []
+
+        # object to store all the files in the package
         self.file_data = {}
 
         self._load_data()
 
     def _load_data(self) -> None:
         """method to load all of the files into a data structure for processing"""
         # walk the tree and add in all the files
+
+        # Try to find a .mpignore file in the directory with the package.mo.
+        # Eventually we should support searching up the tree.
+        project_ignore_file = self.root_directory / '.mpignore'
```

Comment:
so, do you like the name `.mpignore`? I was thinking
* .projectignore (too vague)
* .builderignore (builder is ambiguous)
* .modelicaprojectignore (too long)

ergo

mpignore

thoughts?